### DISCUSSION TITLE
Tag 제공 값 결정하기

### DISCUSSION THREAD

**Discussion by jaejae-yoo** [woowacourse-teams/2022-moamoa#131] [회의 필요] [answered]
## 요약 

Tag 값을 정해서 사용자에게 제공하기로 했기 때문에, 제공할 태그 값들을 다 같이 정할 필요성이 있어보입니다.

## 문제 상황
mysql> select * from tag;
+----+---------+-----------------------+-------------+
| id | name    | description           | category_id |
+----+---------+-----------------------+-------------+
|  1 | BE      | 백엔드                |           2 |
|  2 | FE      | 프론트엔드            |           2 |
|  3 | 4기     | 우테코4기             |           1 |
|  4 | 3기     | 우테코3기             |           1 |
|  5 | Java    | 자바                  |           3 |
|  6 | React   | 리액트                |           3 |
|  7 | Spring  | 스프링                |           3 |
|  8 | Network | 네트워크              |           3 |
|  9 | TS      | 타입스크립트          |           3 |
| 10 | Review  | 코드리뷰              |           3 |
| 11 | Alg     | 알고리즘              |           3 |
| 12 | JPA     | jpa                   |           3 |
| 13 | TDD     | 테스트주도개발        |           3 |
| 14 | Book    | 독서                  |           3 |
| 15 | Health  | 운동                  |           3 |
| 17 | CS      | 컴퓨터 과학           |           3 |
| 18 | Kotlin  | 코틀린                |           3 |
+----+---------+-----------------------+-------------+

현재 데이터베이스 tag 테이블에 저장된 값들 인데요.
여기서 **Network와 CS가 각각 다른 태그로 있는 것이 혼란스러운 것 같습니다.**

## 혼란스러운 이유 및 해결책 의견:
➔  1) Network는 CS에 포함되는 태그가 아닐까? 👉 **Network 태그를 없애고 CS 하나만 둔다.**
➔  2) Network와 CS가 분리되어 있다면, OS, Database 등 다른 값들도 넣어주는 것이 맞지 않을까?  👉 **Network, OS, Database, Algorithm, Datastructure 등의 태그도 포함시킨다.**

팀원분들은 어떻게 생각하시나요 ?

**Comment by tco0427** [comment, marked as answer]
cs보다는 network, os, db, alg 로 나누는게 나을 것 같네요!!

**Reply by tco0427** [reply]
데모데이터여서 이렇게 되었던것 같네요…😅
